### ISSUE TITLE
No tests executed when empty comment in testpackage specification

### ISSUE THREAD

**Issue by safokkens** [utPLSQL/utPLSQL#579]
One of our developers used empty comments in the code, like:

--
-- some message
--

He used this construction in a test package specification and utPLSQL couldn't recognize it as a test package. After a while we figured out that the problem was in the empty comments so we have an easy workaround.

Maybe something that can be fixed some day?

**Comment by jgebal** [comment]
Can you give an isolated tests case (sample package spec) to reproduce?
What version of utPLSQL are you on?
Was the package simply skipped?
How do you know the package?

**Comment by safokkens** [comment]
I'm using v3.0.4. Indeed the package was skipped, it wasn't mentioned in the totals of the test report. I don't know what you mean with "How do you know the package?"

This is what the spec looks like:
CREATE OR REPLACE PACKAGE xxtest_alg 
AS
  --
  -- %suite(xxtool_alg)
  --
  -- %test (GENEREER_BETALINGSKENMERK)
  PROCEDURE bepaal_betalingskenmerk_ok;
  --
END xxtest_alg;
/

**Comment by jgebal** [comment]
Sorry I'm writing from mobile so some odd things going with auto-correction.
Thanks for the example.
Smells like a bug.

**Comment by jgebal** [comment]
I did reproduce that issue on `v.3.0.4`

The current develop branch does not have that issue anymore though.

That means the next release of utPLSQL will not face that issue.
This is because of the rework that was done to annotations.

To avoid regression in future however, we should probably add this test case to set of tests for `ut_sute_manager`

```sql
CREATE OR REPLACE PACKAGE xxtest_alg 
AS
  --
  -- %suite(xxtool_alg)
  --
  -- %test (GENEREER_BETALINGSKENMERK)
  PROCEDURE bepaal_betalingskenmerk_ok;
  --
END xxtest_alg;
/

CREATE OR REPLACE PACKAGE body xxtest_alg 
AS
  PROCEDURE bepaal_betalingskenmerk_ok IS
  BEGIN
    ut.expect(1).to_equal(2);
  END;
  --
END xxtest_alg;
/

exec ut.run('xxtest_alg');
```
The test should say:
**Finds a suite,  when `--%suite` annotation defined after comment line(s) in package spec.**

**Comment by safokkens** [comment]
Thank you for your quick response!

**Comment by jgebal** [comment]
It's resolved in develop - will be part of 3.1.0
```sql
exec ut_runner.purge_cache(user,'PACAKGE');
create or replace package my_test_package
as
   --
   -- %suite
   --
   -- %test
   procedure run_test;
end;
/
create or replace package body my_test_package
as
   procedure run_test is
   begin
     ut3.ut.expect(1).to_equal(0);
   end;
end;
/

set serverout on 
begin ut3.ut.run('my_test_package'); end;
/
```
gives:
```
my_test_package
  run_test [.007 sec] (FAILED - 1)
 
Failures:
 
  1) run_test
      Actual: 1 (number) was expected to equal: 0 (number) 
      at "UT3.MY_TEST_PACKAGE", line 5 ut3.ut.expect(1).to_equal(0);
      
       
Finished in .008238 seconds
1 tests, 1 failed, 0 errored, 0 disabled, 0 warning(s)
```
